src/pages/Possibility/KivanSlider.vue

```diff
@@ -0,0 +1,371 @@
+			});
+		},
+		nextSlide() {
+			this.transitionName = 'kv-slide-left';
+			this.$nextTick(() => {
+				this.$refs.KvCarousel.advance();
+			});
+		},
+		slideTransition(payload) {
+			this.currentIndex = payload;
+		},
+		setSlide(index) {
+			this.$refs.KvCarousel.goToSlide(index);
+		}
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import "settings";
+
+$cta-color: #611b1f;
+$cta-color2: #00244e;
+$cta-color3: #02582e;
+
+.kv-carousel {
+	margin-bottom: rem-calc(140);
+
+	.left-arrow,
+	.right-arrow {
+		z-index: 10000;
+		stroke: $white;
+		height: rem-calc(30);
+		width: rem-calc(30);
+		position: absolute;
+		top: 29%;
+		cursor: pointer;
+
+		@include breakpoint(medium) {
+			top: 33%;
+		}
+	}
+
+	.left-arrow {
+		transform: rotate(90deg);
+	}
+
+	.right-arrow {
+		right: 0;
+		transform: rotate(-90deg);
+	}
+
+	.bullet-wrap {
+		z-index: 10000;
+		position: relative;
+		margin: 0 auto;
+		width: 100%;
+
+		.bullet {
+			width: rem-calc(15);
+			height: rem-calc(15);
+			border: 1px solid $white;
+			border-radius: 50%;
+			display: inline-block;
+			margin: rem-calc(2);
+		}
+
+		.bullet.active {
+			background-color: $white;
+		}
+	}
+
+	.kv-carousel-slide {
+		width: 100%;
+		background: $white;
+	}
+
+	.small-image {
+		margin-bottom: 2rem;
+	}
+
+	.headline {
+		color: $cta-color;
+		font-weight: bold;
+		max-width: 24rem;
+		margin: 0 auto 1rem auto;
+		text-align: center;
+
+		@include breakpoint(large) {
+			margin: unset;
+			margin-bottom: 1rem;
+			text-align: unset;
+		}
+	}
+
+	.subheadline {
+		font-weight: $global-weight-highlight;
+		max-width: 24rem;
+		margin: 0 auto 1.5rem auto;
+		text-align: center;
+
+		@include breakpoint(large) {
+			margin: unset;
+			margin-bottom: 2rem;
+			text-align: unset;
+		}
+	}
+
+	.action-button-wrapper {
+		text-align: center;
+
+		@include breakpoint(large) {
+			text-align: unset;
+		}
+
+		.action-button {
+			background-color: $cta-color;
+			box-shadow: 0 2px darken($cta-color, 10%);
+
+			&:hover,
+			&:focus {
+				background-color: darken($cta-color, 10%);
+			}
+		}
+	}
+
+	/* stylelint-disable */
+	// Slide 2 color override
+	.slide-1 {
+		.headline {
+			color: $cta-color2;
+		}
+
+		.action-button-wrapper .action-button {
+			background-color: $cta-color2;
+			box-shadow: 0 2px darken($cta-color2, 10%);
+
+			&:hover,
+			&:focus {
+				background-color: darken($cta-color2, 10%);
+			}
+		}
+	}
+
+	// Slide 3 color override
+	.slide-2 {
+		.headline {
+			color: $cta-color3;
+		}
+
+		.action-button-wrapper .action-button {
+			background-color: $cta-color3;
+			box-shadow: 0 2px darken($cta-color3, 10%);
+
+			&:hover,
+			&:focus {
+				background-color: darken($cta-color3, 10%);
+			}
+		}
+	}
+	/* stylelint-enable */
+
+	// This is brute force solution to the varing heights of the content within this component
+	// In the furture slideshows should have a defined height, so we can avoid this customization.
+	.slide-placeholder {
+		padding-bottom: 14rem;
```

Comment:
Obvs not ideal but we can refactor later.

Reply:
Creating a refactor ticket to handle early next week. https://kiva.atlassian.net/browse/CASH-1639